include/bl_storage.h

```diff
@@ -50,6 +53,7 @@ u32_t num_public_keys_read(void);
  * @retval -EINVAL  Key has been invalidated.
  * @retval -ENOMEM  The provided buffer is too small.
  * @retval -EFAULT  key_idx is too large. There is no key with that index.
+ * @retval -EHASHFF If any key data contains 0xFFFF, which is unsupported.
```

Comment:
Should we document this requirement? I imagine the case where a customer generates an invalid key and puts it on the device, should they be informed in the docs that they need to watch out for this and throw away invalid key data?

Reply:
Added a note to the bootloader README.